operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -697,42 +705,76 @@ public void dispatchClusterWatch(Watch.Response<ClusterResource> item) {
   }
 
   private void handleAddedCluster(ClusterResource cluster) {
-    getExistingDomainPresenceInfoForCluster(cluster.getNamespace(), cluster.getMetadata().getName()).forEach(info -> {
-      LOGGER.info(MessageKeys.WATCH_CLUSTER, cluster.getClusterName(), info.getDomainUid());
-      createMakeRightOperation(info)
-          .interrupt()
-          .withExplicitRecheck()
-          .withEventData(new EventData(EventItem.CLUSTER_CREATED).resourceName(cluster.getClusterName()))
-          .execute();
-    });
+    List<DomainPresenceInfo> hostingDomains =
+        getExistingDomainPresenceInfoForCluster(cluster.getNamespace(), cluster.getMetadata().getName());
+    if (hostingDomains.isEmpty()) {
+      LOGGER.info(MessageKeys.WATCH_CLUSTER_WITHOUT_DOMAIN, cluster.getMetadata().getName());
+      createMakeRightOperationForClusterEvent(EventItem.CLUSTER_CREATED, cluster).execute();
```

Comment:
I wonder if we may trigger a make-right for some stranded cluster to generate the CLUSTER events and make-right might use some stale DPI.

Reply:
Can you elaborate on the benefit of doing that?

Reply:
Sorry, I was not suggesting to do that but meant to ask a question. We can discuss this in the offline slack thread.